### ISSUE TITLE
[Local Nuget] The V2 feed at ...   returned an unexpected status code '404 Not Found'.

### ISSUE THREAD

**Issue by rohit21agrawal** [NuGet/NuGetGallery#5370]
@LightVolk commented on [Mon Jan 29 2018](https://github.com/NuGet/Home/issues/6494)

## Details about Problem
Visual Studio 2017 Enterprise 15.5.5.

NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe):
VS UI
NuGet version (x.x.x.xxx):
NugetServer 2.14.2  (also do not work at 3.0.2 too)

VS version (if appropriate):
Visual Studio Enterprise 2017 15.5.5
OS version (i.e. win10 v1607 (14393.321)):
Win7 Pro SP 1

Worked before? If so, with which NuGet version:
No
## Detailed repro steps so we can see the same problem

1.I use this instruction: [https://docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server](url) 
2.Create  Empty Web Site (.net 4.6)
3. Install Nuget.Server 2.14.2 on site.  (also, do not work at 3.0.2)
4. On Packages folder add some nuget packages (do not set any additional properties such as 'Build Action->Content and Copy to Output'
5. Run site locally in VS -> Ctrl-F5

Here result:
![111](https://user-images.githubusercontent.com/1236916/35506870-e0d4647e-04fb-11e8-86af-a01d5dd7caef.JPG)

6. So i go to VS console project and add new nuget source [http://localhost:63289/nuget](url) to settings. Turn off nuget.org source.

7. Try to find any package, but i get error:

`[Local Nuget] The V2 feed at 'http://localhost:63289/nuget/Search()?$filter=IsLatestVersion&searchTerm=''&targetFramework='net46'&includePrerelease=false&$skip=0&$top=26&semVerLevel=2.0.0' returned an unexpected status code '404 Not Found'.`

7.1 Also at Web.config set: 
     ` 
    <add key="packagesPath" value="d:\Nuget\nuget\" />`
7.2 Site works on IIS 10 Express.

## Other suggested things
8. Also, when i try to view packages on server (click on link "here" at site) i get another error:
![2](https://user-images.githubusercontent.com/1236916/35507006-658606b4-04fc-11e8-89c2-ae883c48c9f3.JPG)
  Error 404 on Russian language.
### Verbose Logs
If requested.

### Sample Project
I upload project to my Google Drive. It is over 49 mb.
[https://drive.google.com/file/d/1MIcPbBJvpRaOxrbnHc8VTd3MHQsS47WI/view?usp=sharing](url)

**Comment by skofman1** [comment]
This was opened a while ago. We must have missed it. Please reopen if assistance is still needed.

**Comment by Bloodcoder** [comment]
@skofman1 , if using Gallery.StorageType with a value of FileSystem in NugetGallery is there another setting to say where the files should be stored?

**Comment by skofman1** [comment]
Yes. It's in web.config and called Gallery.FileStorageDirectory

**Comment by Bloodcoder** [comment]
> Yes. It's in web.config and called Gallery.FileStorageDirectory

Okay, so, that is set and when I upload a package it shown in the search page using chrome. I checked the folder where the package files are placed and the test package I published showed up in the folder. The issue that occurs is when using the package manager in visual studio it returns the error 

[Package source] The V2 feed at 'https://myazurehostedsite.azurewebsites.net/nuget/Search()?$filter=IsLatestVersion&searchTerm=''&targetFramework=''&includePrerelease=false&$skip=0&$top=26&semVerLevel=2.0.0' returned an unexpected status code '404 Not Found'.

I pulled latest release from github and I also tried the most recent code from master.

Maybe it is a nuget issue.
